pkg/corerp/frontend/controller/environments/getrecipemetadata_test.go

```diff
@@ -74,7 +74,14 @@ func TestGetRecipeMetadataRun_20231001Preview(t *testing.T) {
 				"mongodbName":    map[string]any{"type": "string"},
 			},
 		}
-		mEngine.EXPECT().GetRecipeMetadata(ctx, recipeDefinition).Return(recipeData, nil)
+		mEngine.EXPECT().GetRecipeMetadata(ctx, engine.GetRecipeMetadataOptions{
```

Comment:
It is great to see that we are using actual expected objects instead of `gomock.Any()`s. Thanks!